packages/core/src/icons/ui/icon-shopping-cart.css

```diff
@@ -0,0 +1,6 @@
+@svg-load icon url(../../svg/ui/shopping-cart.svg);
+
+.hds-icon--heart {
```

Comment:
Should it be .hds-icon--shopping-cart? Not heart?

Reply:
- [ ] Fix copy-paste error